pom.xml

```diff
@@ -15,6 +15,13 @@
             <version>24.0-jre</version>
         </dependency>
 
+        <!-- https://mvnrepository.com/artifact/org.apache.commons/commons-math3 -->
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-math3</artifactId>
+            <version>3.0</version>
```

Comment:
why not to use the latest one? 3.6.1? 3.0 is from 2003:)

Reply:
I'm really sorry. Upgraded the version.